cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
+= Notable features in the previous release
+
+{odo-title}{nbsp}{odo-ver} offers a simple way to develop applications on {product-title}. {odo-title} is completely client-based and requires no server within {product-title} cluster.
+
+{odo-title}{nbsp}{odo-ver} comes with:
+
+* Simple syntax and design centered around concepts familiar to developers such as projects, applications and components.
+* Compatibility with any language or runtime within the OpenShift catalog of component types.
+* Capability to add custom component types using custom image builders.
+* Official support for Java and Node.js component types.
+* Service catalog which allows users to connect applications deployed on {product-title} cluster to the Template Service Broker.
+* `odo service create` - an interactive mode which guides users through the process of creating applications.
+* `odo watch` - a feature that automatically detects changes to local code and applies the changes to the component in real time.
+
+////
+
+[id="odo-getting-support_{context}"]
+== Getting support
+
+.For Documentation
+
+If you have found an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+
+.For Product
+
+If you have found an error, encountered a bug or have suggestions for improving the functionality of {odo-title}, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *Red Hat odo for OpenShift Container Platform* product type.
+
+Provide as many details in the description of the issue as possible.
+
+[id="odo-fixed-known-issues_{context}"]
+== Fixed known issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760573[Bug 1760573] An active pointer is not pointing to a current active project.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760578[Bug 1760578] `odo watch` does not fail with error message for git based components but it should.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760583[Bug 1760583] `odo config unset` does not unset an environment variable but says it did.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760585[Bug 1760585] `odo delete --all` deletes the`$HOME/.odo` folder when run from `$HOME`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760589[Bug 1760589] Autocompletion does not work for  `--context` in `odo push`.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1761442[Bug 1761442] `component create` fails with flag `--context` and `--binary` if the binary is located in a temporary folder.
+
+[id="odo-known-issues_{context}"]
+== Known issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760574[Bug 1760574] A deleted namespace is being listed in the `odo project get` command.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760575[Bug 1760575] `odo app delete` removes application components but not services.
+
```

Comment:
Please change to this format in all issues that start with a command: The `...` command